### PR TITLE
psp-2975 reorganize psp folders.

### PR DESCRIPTION
**Pull request by devinleighsmith**
-remove adr
-move script related folders into /tools
-remove unused testing folders
-move geoserver to /tools
update tooling to point to new locations.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -101,24 +101,24 @@ endif
 ##############################################################################
 .PHONY: version
 version: require-node ## Outputs the latest released version
-	@build/version.sh
```
Comment (devinleighsmith) [inline Makefile:104]: I've tested all of the changed commands here to confirm they work still.

File: docker-compose.yml [hunk 2]
```diff
@@ -62,11 +62,11 @@ services:
   geoserver:
     image: kartoza/geoserver:2.19.2
     volumes:
-      - ./geoserver/geoserver_data:/opt/geoserver/data_dir/workspaces
+      - ./tools/geoserver/geoserver_data:/opt/geoserver/data_dir/workspaces
```
Comment (devinleighsmith) [inline docker-compose.yml:65]: manually tested this as well.

**Comment by devinleighsmith** [comment]
Haven't tested any of the actions to make sure they weren't affected by the move. Can try on my fork if we have concerns.

File: build/bump-version.js [hunk 3]
```diff
@@ -1,142 +0,0 @@
-const fs = require('fs');
```
[the remark this answers is not in the document]
Comment (devinleighsmith) [reply build/bump-version.js]: fixed, issue was a gitignore in the cicd folder.

**Review by asanchezr — APPROVED** [review]

**Review by FuriousLlama — APPROVED** [review]

**Pull request merged**